shuup/gdpr/providers.py

```diff
@@ -67,3 +76,23 @@ class GDPRCheckoutFieldProvider(GDPRFieldProvider):
 
 class GDPRAuthFieldProvider(GDPRFieldProvider):
     error_message = _("You must accept to this to authenticate.")
+
+    def get_fields(self, **kwargs):
+        request = kwargs.get("request", None)
+        gdpr_settings = get_gdpr_settings(request)
+        if not gdpr_settings:
+            return []
+
+        if gdpr_settings.skip_consent_on_auth:
+            return [
+                FormFieldDefinition(
+                    name=None,
+                    field=forms.CharField(
+                        initial=gdpr_settings.safe_translation_getter("auth_consent_text"),
```

Comment:
when I submit the form and it contains errors, None is rendered here
![image](https://user-images.githubusercontent.com/8770370/58252056-92be5380-7d3b-11e9-90a6-4ac9523064c2.png)